spring-boot/src/main/resources/application.properties

```diff
@@ -6,7 +6,6 @@ spring.datasource.password=${POSTGRESQL_DB_PASSWORD:admin}
 spring.datasource.driver-class-name=org.postgresql.Driver
 
 spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
Please use the Jakarta Persistent property instead.

Reply:
Done. See:

- https://www.baeldung.com/spring-data-jpa-generate-db-schema
- https://jakarta.ee/specifications/persistence/3.2/jakarta-persistence-spec-3.2